evaluator/perception_online_evaluator/src/metrics_calculator.cpp

```diff
@@ -399,6 +402,35 @@ MetricStatMap MetricsCalculator::calcYawRateMetrics(const ClassObjectsMap & clas
   return metric_stat_map;
 }
 
+MetricStatMap MetricsCalculator::calcObjectsCountMetrics() const
+{
+  MetricStatMap metric_stat_map;
+
+  for (const auto & [label, count] : historical_detection_count_map_) {
+    Stat<double> stat;
+    stat.add(
+      static_cast<double>(static_cast<double>(count) / static_cast<double>(objects_count_frame_)));
+    metric_stat_map["historical_objects_count_" + convertLabelToString(label)] = stat;
+  }
+
+  int vector_size = detection_count_vector_.size();
+  DetectionCountMap interval_detection_count_map = initializeDetectionCountMap();
+
+  for (const auto & [detection_count_map, stamp] : detection_count_vector_) {
+    for (const auto & [label, count] : detection_count_map) {
+      interval_detection_count_map[label] += count;
+    }
+  }
+
+  for (const auto & [label, count] : interval_detection_count_map) {
+    Stat<double> stat;
+    stat.add(static_cast<double>(count) / static_cast<double>(vector_size));
+    metric_stat_map["interval_objects_count_" + convertLabelToString(label)] = stat;
+  }
+
+  return metric_stat_map;
+}
```

Comment:
can be written like following? e.g. interval_detection_count_map is not neccesaray

```suggestion
MetricStatMap MetricsCalculator::calcObjectsCountMetrics() const
{
  MetricStatMap metric_stat_map;

  for (const auto & [label, count] : historical_detection_count_map_) {
    metric_stat_map["historical_objects_count_" + convertLabelToString(label)].add(
      static_cast<double>(count) / static_cast<double>(objects_count_frame_));
  }

  const double frame_count = static_cast<double>(detection_count_vector_.size());
  for (const auto & [detection_count_map, _] : detection_count_vector_) {
    for (const auto & [label, count] : detection_count_map) {
      metric_stat_map["interval_objects_count_" + convertLabelToString(label)].add(
        static_cast<double>(count) / frame_count);
    }
  }

  return metric_stat_map;
}
```

Reply:
yes!